test/development/acceptance/hydration-error.test.ts

```diff
@@ -4,9 +4,14 @@ import { FileRef, nextTestSetup } from 'e2e-utils'
 import { outdent } from 'outdent'
 import path from 'path'
 
-describe('Error overlay for hydration errors', () => {
+// TODO: Enable this test once react 18 is supported for pages router
+describe.skip('Error overlay for hydration errors', () => {
   const { next } = nextTestSetup({
     files: new FileRef(path.join(__dirname, 'fixtures', 'default-template')),
+    dependencies: {
+      react: 'latest',
+      'react-dom': 'latest',
```

Comment:
should this explicitly target 18? I imagine in the future `latest` will become 19 and then this test will start failing